game/scripts/npc/abilities/phantom_lancer_phantom_edge.txt

```diff
@@ -37,7 +37,7 @@
 			"04"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"bonus_agility"		"6 12 18 24 36 75"
+				"bonus_agility"		"6 14 22 30 36 75"
```

Comment:
following 2x -> 6x, then scaling down slightly because agility is OP

`6 14 22 30 45 90`